test/integration/serving-files.spec.js

```diff
@@ -62,6 +63,30 @@ describe('serves', function() {
       .end(done);
   });
 
+  it('cannot access files above the root', function(done) {
+    var app = connect()
+      .use(superstatic(options()));
+
+    request(app)
+      .get('/../README.md')
```

Comment:
Technically, would `/dir/../index.html` work here? Seems a little odd - I've never seen relative paths like this in URLs